### ISSUE TITLE
Enable GitHub request for read:user scope in production which allows to see if the user has 2FA enabled

### ISSUE THREAD

**Issue by gdestuynder** [mozilla-iam/iam-project-backlog#141]
When enabled this will prompt the users using github for a new authorization: read-only access to your private profile.
It needs to be enabled before we can require MFA for GitHub accounts

**Comment by gdestuynder** [comment]
@gene1wood @hmitsch @m-branson 

This is a user-impacting change (GitHub users will see a new request for authorization), so looking for some comments - though I'd vote to just do it.

If you're curious how it looks like just login with github to https://social-ldap-pwless.testrp.security.allizom.org/

**Comment by mbransn** [comment]
@gdestuynder The impact was light and much as expected, I'm good with implementing.

**Comment by andrewkrug** [comment]
@gdestuynder how many users do we have that don't have MFA in our ORG?

**Comment by hmitsch** [comment]
@gdestuynder I'd also go for it. Maybe a short notice on the Slack #announcements channel to inform people about the user-impacting change?

**Comment by gdestuynder** [comment]
and/or to the iam public mailinglist i'd say

**Comment by hmitsch** [comment]
@gdestuynder ha, excellent point ... what is the "IAM Public Mailinglist" at this point?

I currently know of these lists:
- **iam-dev@mozilla.com:** Some kind of internal (?) mailing list? By the way, I think we wanted to transition this one to Discourse. Could be an opportunity to merge it with the item mentioned below.
- **iam@mozilla-community.org:** Discourse powered, public list.
- **iam+internal@mozilla-community.org:** Discourse powered list, restricted to people on https://mozillians.org/group/iam-project/
- **[mozilla.dev.identity](https://groups.google.com/d/msg/mozilla.dev.identity/)** (Note: I am admin)

**Comment by gdestuynder** [comment]
it looks like we could merge many of these
iam-dev @mozilla.com can redirect to iam+internal@mozilla-community.org
mozilla.dev.identity could go to iam@mozilla-community.org or does it have to stay separated?

**Comment by hmitsch** [comment]
Sorry for hijacking the conversation. I created a separate issue for mailing lists (#149).

**Comment by gdestuynder** [comment]
lets depend this on #149 and I'll switch over when #149 is closed/done

**Comment by gdestuynder** [comment]
communication sent to iam@mozilla-community.org

**Comment by gdestuynder** [comment]
The change has been made (timestamp: this comment)
